### PR TITLE
Jphelps92 bio

### PR DESCRIPTION
**Pull request by jphelps92**
- [x] Include my name
- [x] Be located in the `docs` folder
- [x] Include hometown and occupation

https://github.com/githubteacher/developers-09-24-2015/issues/9

### CODE REVIEW

File: docs/jphelps92.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Username: **jphelps92**
+
+Full name: **Jonathan Phelps**
+
+Hometown: **Brookfield, WI**
```
Comment (HarryPalz) [inline docs/jphelps92.md]: I'M ALSO FROM HERE

File: docs/jphelps92.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Occupation: **Software Engineer at Johnson Controls, Inc.**
+
+I am 23 years old. I graduated in May of 2014 with a Bachelor's degree in Computer Science: Software Engineering from Carroll University. I started working at Johnson Controls in May of 2013 as a Software Engineering Intern. After I graduated, I was offered a full-time position here.
```
Comment (HarryPalz) [inline docs/jphelps92.md:9]: Nice to know, but this portion wasn't a part of the assignment. :u6307:

Comment (jphelps92) [inline docs/jphelps92.md:9]: I thought this was a bio... should I remove it?

File: docs/jphelps92.md (hunk 1 above)
Comment (jphelps92) [inline docs/jphelps92.md]: Yay!

File: docs/jphelps92.md (hunk 2 above)
Comment (HarryPalz) [inline docs/jphelps92.md:9]: You can keep it if you want :melon:

**Comment by jphelps92** [comment]
https://github.com/githubteacher/developers-09-24-2015/pull/30#discussion_r40335975

**Comment by jphelps92** [comment]
:+1:

**Pull request merged**